python_modules/dagster/dagster/components/component/component.py

```diff
@@ -268,7 +268,10 @@ def build_defs(self, context: "ComponentLoadContext") -> Definitions: ...
     def load(cls, attributes: Optional[BaseModel], context: "ComponentLoadContext") -> Self:
         if issubclass(cls, Resolvable):
             context_with_injected_scope = context.with_rendering_scope(
-                {"component_tree": context.component_tree}
+                {
+                    "load_component_at_path": context.component_tree.load_component_at_path,
+                    "build_defs_at_path": context.component_tree.build_defs_at_path,
+                }
```

Comment:
We definitely should add a linear issue to make these more self-documenting and discoverable to users. @benpankow can you add an issue for that?

Reply:
https://linear.app/dagster-labs/issue/ADOPT-1645/make-base-udfstemplate-vars-more-discoverable